csfieldguide/chapters/content/en/big-data/sections/the-whole-story.md

```diff
@@ -0,0 +1,10 @@
+# The Whole Story
+
+Big Data gives us an opportunity to gain a much deeper understanding of how many things in the world around us work. It is a new and exciting field, and even with the huge amounts of work and research happening at the moment we are barely scratching the surface of the possibilities. It has very close ties with Artificial Intelligence, as many AI systems use Big Data, and a lot of Big Data analysis is done using AI algorithms.
+
+It is so new and influential in fact that there is now a [huge demand](https://economicgraph.linkedin.com/resources/linkedin-workforce-report-august-2018) for those with Data Science skills. Data Scientists are people with the skills and knowledge to appropriately analyse Big Data. With the amount of data we are collecting each day growing rapidly, it’s not surprising that this field is also rapidly growing.
+
+But not only is there a huge demand for experts in this field, there is also demand for innovation. We are already pushing the limits of the infrastructure currently supporting Big Data.
+For example, once it is completed it is estimated that the [Square Kilometre Array](https://www.skatelescope.org/the-ska-project/) telescope will produce [160 terabytes](https://www.skatelescope.org/frequently-asked-questions/) of raw data every second, which is over 10 exabytes of data being produced everyday! Which is much, much more data than currently exists on the entire internet.
```

Comment:
Maybe have the second link encompass all of 'will produce 160 terabytes of raw data every second' so that it doesn't look like it's some definition for that number of terabytes